docs/topics/proc-getting-started-update-booster.adoc

```diff
@@ -0,0 +1,37 @@
+[[update-booster]]
+= Updating Your Booster and Deploying the Changes
+To push changes to this booster running in {OpenShiftOnline} or your {OpenShiftLocal}, you can push a new commit to the GitHub repository and start a new build.
+
+[sidebar]
+.Prerequisites
+--
+* Have a booster deployed to xref:oso-create-booster[{OpenShiftOnline}] or xref:osl-create-booster[{OpenShiftLocal}] using the _Deploy in OpenShift Online_ option.
+--
+
+.Procedure
+. Clone your project from GitHub:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ git clone git@github.com:USERNAME/PROJECT-NAME.git
+----
+
+. Open your project in your desired IDE or editor, such as xref:use_devstudio[JBDS].
+. Make an update to the project.
+. Save your changes.
+. Commit and push your changes back to GitHub:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ git add .
+$ git commit -m "Made an update"
+$ git push
+----
+
+. Navigate back to the {OpenShiftOnline} or {OpenShiftLocal} Web console and ensure the project of your mission is selected.
```

Comment:
I've seen "Web Console" somewhere else in this PR, and it's "Web console" here. I guess it could also be "web console". What's the intended capitalization here?

Reply:
I believe we use "Web console". Nice catch here